CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add autocomplete feature when inputting admin name for contribution catalog.[#665](https://github.com/rokwire/rokwire-building-blocks-api/issues/665)
 - Add search endpoints using capability and talent id.[#662](https://github.com/rokwire/rokwire-building-blocks-api/issues/662)
 - Add release script for Contributions Building Block.[#675](https://github.com/rokwire/rokwire-building-blocks-api/issues/675)
+- Add okay endpoint to Contributions Building Block.[#670](https://github.com/rokwire/rokwire-building-blocks-api/issues/670)
```

Comment:
```suggestion
- Add okay endpoint to Contributions Building Block.[#679](https://github.com/rokwire/rokwire-building-blocks-api/issues/679)
```

Reply:
Oh, I made my branch name wrong from the first place. Sorry, changed it.

Reply:
No issues. Thanks.